### ISSUE TITLE
Move alpha into grouping expression for ridge regression fit

### ISSUE THREAD

**Issue by karenfeng** [projectglow/glow#289]
During `RidgeRegression.fit()`, we group on `['sample_block', 'label']` but not on `'alpha'`. This becomes a limiting factor on our scalability due to PyArrow limits, which constrain the number of 8-byte float values in a vector to 132,152,839; see the limits described in https://github.com/projectglow/glow/pull/282.

**Comment by henrydavidge** [comment]
@karenfeng did we end up doing this one?

**Comment by karenfeng** [comment]
No, I believe this is unchanged: https://github.com/projectglow/glow/blob/a63306ee307e9209bce0ea3f7f6d74d60bb8ab32/python/glow/wgr/linear_model/ridge_model.py#L217
